Dockerfile-acceptance

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/ukhomeofficedigital/nodejs-base:v4.4.2
+FROM node:5
```

Comment:
just used the same as firearms, it _was_ latest when these dockerfiles were written